UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -39,23 +45,11 @@ namespace MLAgents
     /// </remarks>
     [HelpURL("https://github.com/Unity-Technologies/ml-agents/blob/master/" +
         "docs/Learning-Environment-Design-Academy.md")]
-    public abstract class Academy : MonoBehaviour
+    public class Academy : MonoBehaviour
```

Comment:
should we make this `sealed` or is that overkill?

Reply:
I think it would clearly communicate that the Academy is not to be subclassed anymore.  It makes sense to add `sealed` to me.

Reply:
Doh, forgot to add this. If for some reason, I don't get the singleton implemented before the next release, I'll seal it for that.